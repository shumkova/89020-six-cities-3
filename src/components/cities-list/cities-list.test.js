import React from "react";
import renderer from "react-test-renderer";
import CitiesList from "./cities-list";

const cities = [`Amsterdam`, `Moscow`, `New York`, `London`];
const activeCity = `London`;
const onCityClick = () => {};

it(`CitiesList should render correctly`, () => {
  const tree = renderer.create(
      <CitiesList
        cities={cities}
        activeCity={activeCity}
        onCityClick={onCityClick}
      />
  ).toJSON;

  expect(tree).toMatchSnapshot();
});
