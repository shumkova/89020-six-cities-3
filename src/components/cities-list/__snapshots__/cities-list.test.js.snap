// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CitiesList should render correctly 1`] = `[Function]`;
